CHANGES.rst

```diff
@@ -2,8 +2,15 @@
 Changes
 =======
 
-Unreleased
-==================
+`Unreleased <https://github.com/bird-house/threddsclient/tree/master>`_
+==========================================================================================
```

Comment:
Is it possible to make a variable-length underline here to match the length of the line above? The main reason being that this will emit warnings when building the docs with Sphinx.

I know this is easy to do with jinja...

Reply:
This cannot be defined natively for `bump2version` config, as far as I know. It's possible to do some envvar-templating with shell replacements beforehand, but it makes the tool less user-friendly to employ.

The length can be anything, as long as it remains longer or equal to the text above to avoid RST not detecting it as a valid header. However, this is ~90 chars currently and other RST + code sections go beyond that already. If this is flagged somehow, it is probably better to adjust to `line-lenght=120` with the relevant option of the tool that flags it.